export const signup = async() => {
    try {
        console.log("worker controller");
        
    } catch (error: any) {
        console.log(error);
        
    }
}